packages/http/src/models/Transaction.ts

```diff
@@ -259,20 +259,20 @@ function prepareTxn(txn: any) {
     txn.fee = new Balance(txn.fee, CurrencyType.dataCredit)
   }
 
-  if (txn.stakingFee && typeof txn.stakingFee === 'number') {
-    txn.stakingFee = new Balance(txn.stakingFee, CurrencyType.dataCredit)
+  if (txn.staking_fee && typeof txn.staking_fee === 'number') {
```

Comment:
Ah whoops we should have noticed this in review. Can we add maybe a `AnyHTTPTransaction` type to txn? I guess its kinda hard to type it since it could be any of the HTTP response ones?